### ISSUE TITLE
[enhancement] give a list of valid id for PREFERRED_REGION

### ISSUE THREAD

**Issue by solsticedhiver** [pia-foss/manual-connections#117]
Documentation about the list of region id to use with PREFERRED_REGION would be good.

I had to download `https://serverlist.piaservers.net/vpninfo/servers/v4|jq` and use `curl https://serverlist.piaservers.net/vpninfo/servers/v4|jq .` to find the correct id I wanted.

Because it needs the proper capitalisation, otherwise this fails (because of the server)

**Comment by g00nix** [comment]
I am not sure I understand the request. Can you please detail the issue you are facing?

**Comment by solsticedhiver** [comment]
If I want to run `./get_regions.sh` alone on the cli, I give it PREFERRED_REGION as an env var. What are the value I am expecting to give in the varaible ?

I tried 'France' and it failed because it was 'france' that was expected. For South-Africa, it's 'za' not (south-africa or South-Africa or ZA)

**Comment by solsticedhiver** [comment]
May be provide an option in the script to show the output of something like:

     curl https://serverlist.piaservers.net/vpninfo/servers/v4  |jq '.regions[]| .id'|sed 's/"//g'|less

It would be more flexible than a hard-coded list.

**Comment by faireOwl** [comment]
The script provides a list of the servers, as well as output clarifying where they are located. The run_setup.sh script will walk you through the process and provide output specifically intended to clarify the script function for you, be sure to read the output.

A list of servers and connection details, ordered by latency can be found in at : /opt/piavpn-manual/latencyList

That is specifically provided for anyone preferring to get details for one line calls.

**Comment by faireOwl** [comment]
There is also a video guide available: https://www.youtube.com/watch?v=Sq8T7oBog44

**Comment by solsticedhiver** [comment]
I did not use `run_setup.sh`. I did not see the benefit of it.

OK. May be, I fast read the README. May be just rewrite some section to make it more obvious ?

Like, I need to run `.get_regions.sh` first without PREFERRED_REGION st or with AUTOCONNCEC=true, then look at the list in /opt/piapvn-manual/latencyList, then rerun with my preferred region id ?

The README becomes quite big. May be split it into more files. And to be able to explain som ethings better ?
